src/main/java/reposense/system/LogsManager.java

```diff
@@ -88,7 +91,7 @@ private static void removeHandlers(Logger logger) {
      * Creates {@link FileHandler} if it is null.
      */
     private static void addFileHandler(Logger logger) {
-        Path path = logFolderLocation.resolve(LOG_FOLDER_NAME);
+        Path path = TEMP_LOG_DIR.resolve(LOG_FOLDER_NAME);
 
         try {
             if (!Files.exists(path)) {
```

Comment:
Is this check still relevant? Considering we use the current working directory as per TEMP_LOG_DIR

Reply:
Good catch, thanks for noticing this

Reply:
I left the creation of the directory as it is still required since the temp folder might not exist. However,  I have shifted the logging to below, after the addition of the `fileHandler` to the logger, so that the logs will be output correctly.